filters/src/main/scala/org/example/clumpzootsample/Filters.scala

```diff
@@ -11,4 +13,15 @@ object Filters {
         next(request)
       }
     }
+
+  val exceptionLogFilter =
+    new Filter {
+      override def apply(request: Request, next: Service) = {
+        val response = next(request)
+        response.onFailure {
+          case _ => println(s"Exception was thrown!")
```

Comment:
Zoot doesn't propagate exceptions, it transforms then into failed responses. This allows to encode/decode exceptions between client/server without loosing information